evm-contracts/src/v0.8/interfaces/KeeperCompatibleInterface.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity ^0.8.0;
```

Comment:
Can you please put this file in the `contracts/` directory instead of `evm-contracts/`?